config/database.yml

```diff
@@ -18,7 +18,7 @@ default: &default
   adapter: mysql2
```

Comment:
this file should not be committed and pushed.
there's an entry in `.gitignore` file to ignore this file in a commit message.
check it out why is it not working